inspirehep/modules/records/cli.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import click
+
+from flask.cli import with_appcontext
+
+from .checkers import check_unlinked_references
+
+
+@click.group()
+def check():
+    """Commands to check different types of data"""
+
+
+@check.command()
+@click.argument('doi_file_name', type=click.File('w'), default='doi_file')
+@click.argument('arxiv_file_name', type=click.File('w'), default='arxiv_file')
+@with_appcontext
+def unlinked_references(doi_file_name, arxiv_file_name):
```

Comment:
Can you provide a meaningful docstring here? this will be shown to the user when doing `inspirehep check unlinked_references --help`.